NickvisionTubeConverter.Shared/Resources/Strings.resx

```diff
@@ -442,4 +478,12 @@
 	<data name="SaveWarning.GTK" xml:space="preserve">
 		<value>The selected save folder is outside the Flatpak sandbox. The file will be saved, but the folder path will not be remembered for future use.&#xA;&#xA;If you want to always remember this folder, give Tube Converter Flatpak write permissions to the selected folder. (You can use Flatseal to achieve this task)</value>
 	</data>
+
+	<data name="AllowBackground.GTK" xml:space="preserve">
+		<value>Allow running in the background</value>
```

Comment:
Could we make these non-GTK only strings? I'd like to implement this switch on WinUI too as most applications allow users to turn off the system-tray icon if they choose.